right/src/mouse_controller.c

```diff
@@ -213,7 +213,7 @@ static float computeModuleSpeed(float x, float y, uint8_t moduleId)
     }
 
     float normalizedSpeed = *currentSpeed/midSpeed;
-    return moduleConfiguration->speed*(float)pow(moduleConfiguration->acceleration * normalizedSpeed, accelerationExp);
```

Comment:
@mondalaci if you have been trying to fine tune the acceleration and did not feel any effect, this is why.

Namely, lets rewrite the (left) side in a mathematical notation:
```speed * (acceleration * x) ^ accelerationExp```. 
But this can also be rewritten as 
```speed * (acceleration ^ accelerationExp) * (x ^ accelerationExp)```
 Since both `acceleration` and `accelerationExp` are constants... : 
```
[ speed * (acceleration ^ accelerationExp) ] * [ (normalize(x) ^ accelerationExp) ]
[           this controlls speed           ] * [    this controls acceleration    ]
```